engineering/conventions.md

```diff
@@ -0,0 +1,47 @@
+
+# Development Conventions
+
+## Introduction
+
+This guide documents development conventions that are language-independent, so they are applicable to every source{d} project.
+
+We have additional conventions for each language:
+
+* [Go](conventions-go.md)
+* [Python](conventions-python.md)
+* [Scala](conventions-scala.md)
+
+## Versioning
+
+* Use [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html).
+* When creating version tags in git, use the `v` prefix (e.g. `v0.5.0`). See our [git guide](git-flow.md) for more information.
+
+## Build System
+
+* All projects have a `Makefile` at the top-level directory.
+* Makefile should be compatible with `[GNU make](https://www.gnu.org/software/make/manual/make.html)`.
+* [src-d/ci](https://github.com/src-d/ci) should be used in these makefiles to provide integration with our continuous integration systems.
+
+## Continuous Integration
```

Comment:
Do you think it might be worth also to clarify "Travis for CI, Drone for CD" approach?

Reply:
It has been discussed the need to move to Travis for CD in public repositories, to avoid heterogeneous repositories